raisinbread/instruments/EEG_Acquisition_form.rules

```diff
@@ -0,0 +1,44 @@
+research_staff_two{-}Not Required{-}research_staff_two{@}=={@}NEVER_REQUIRED
+research_staff_three{-}Not Required{-}research_staff_three{@}=={@}NEVER_REQUIRED
+session_testing_room{-}Not Required{-}session_testing_room{@}=={@}NEVER_REQUIRED
+subject_behavioral_assistant{-}Required if there is a behavioural assistant{-}subject_behavioral_assistant_present{@}=={@}yes
```

Comment:
This `u` is inconsistent --- Please grep aces/Loris for "ehavio" and use the most frequent spelling you find -- it's either behavior* or behaviour*
regis: it's a canadian/us spelling issue - that's why i'm recommending you go with the most frequent spelling loris uses.

Reply:
`behaviour` it is, see comment below